torchgeo/datasets/wms.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# Author: Ian Turton, Glasgow University ian.turton@gla.ac.uk
```

Comment:
We don't normally edit the license header. Your contribution will still be included in the git history though.